test/integration/find_local_council_test.rb

```diff
@@ -368,4 +380,15 @@ class LocalCouncilTest < ActionDispatch::IntegrationTest
       end
     end
   end
+
+  def assert_beta_label(with_link: {})
+    args = {}
+    args[:text] = %{<a href=\\"#{with_link[:href]}\\">#{with_link[:text]}</a>} unless with_link.empty?
+    # components aren't rended in test, but we get the partial arguments as
+    # json on the page to test so we can test that
+
```

Comment:
This seems a bit gross, are there alternatives for testing components?  I'd really like to be able to use `assert page.has_link? with_link[:text], href: with_link[:href]` instead of building a link manually and breaking when we add extra attributes to the link in the implementation.